core/src/edu/asu/c3simulator/screens/ManagePanel.java

```diff
@@ -0,0 +1,54 @@
+package edu.asu.c3simulator.screens;
+
+import com.badlogic.gdx.Game;
+import com.badlogic.gdx.Gdx;
+import com.badlogic.gdx.scenes.scene2d.Stage;
+import com.badlogic.gdx.scenes.scene2d.ui.Skin;
+
+import edu.asu.c3simulator.widgets.NavigationPanel;
+
+/**
+ * Creates a panel that allows for navigation through the management screens.
+ * 
+ * @author Reigel, Justin
+ * 
+ */
+public class ManagePanel
+{
+	private static final int DESIGN_WIDTH = 1280;
+	private static final int DESIGN_HEIGHT = 720;
+	float padding = 0.01f * DESIGN_HEIGHT;
```

Comment:
Are we accessing these in the package? Will we be in the future? 
This design decision is worth a team discussion